IdentityCore/src/oauth2/MSIDOauth2Factory.m

```diff
@@ -229,7 +229,7 @@ - (BOOL)fillAccessToken:(MSIDAccessToken *)token
     
     if (!token.accessToken)
     {
-        MSID_LOG_ERROR(nil, @"Trying to initialize access token when missing access token field");
+        MSID_LOG_WARN(nil, @"Trying to initialize access token when missing access token field");
```

Comment:
Why this is "non-critical"? Don't we require AT to be present in server response?

Reply:
Fixed this one. You're right, this one is not reading it from cache, so should be error level.